### ISSUE TITLE
Mirroring of Android P in AirtestIDE

### ISSUE THREAD

**Issue by nikkitagupta** [AirtestProject/AirtestIDE#302]
Hi I am facing one issue with Android P version , the device is not getting mirrored when I connect any device having Android Pie version, it shows only black screen whenever I connect my device. I thought may be its a IDE version issue so tried using the latest version but still I am getting the same issue. Can anyone help me on this ?

**Comment by yimelia** [comment]
is there any trace or log in AirtestIDE or command line window? like [this](http://airtest.netease.com/docs/en/3_faq/ide_crash.html#how-can-i-check-the-cause-of-airtestide-crash)

**Comment by nikkitagupta** [comment]
No, I am sharing the screenshot of both IDE and command line window.

Thanks
Nikita Gupta
Software Engineer

**Comment by nikkitagupta** [comment]
No, I am sharing the screenshot of both IDE and command line window which I am getting when trying to click **connect** button under Mobile phone connection tab.
Note - My ADB is connected I can see the list of device using command "adb devices"

![ide_screenshot](https://user-images.githubusercontent.com/6149365/50680307-35946080-102d-11e9-8e7a-e5e567f2509f.png)
![logcat_screenshot](https://user-images.githubusercontent.com/6149365/50680308-35946080-102d-11e9-961c-c813207a3ec3.png)

**Comment by yimelia** [comment]
What brand is your mobile phone? maybe we miss some files of minicap.

but there is another way to see the screen, try to click "use javacap" before the `connect` button, and then start connecting your phone

**Comment by yimelia** [comment]
try `use javacap` and `use ADB orientation` ?
and do you have any console log? it maybe help us find the problem

**Comment by nikkitagupta** [comment]
Hi 
Use of Javacap and  ADB orientation also doesn't work. Please help me its hampering my work.

**Comment by yimelia** [comment]
update the AirtestIDE to v1.2.0 and try again?

**Comment by yimelia** [comment]
v0.0.10 doesn't support the android P
